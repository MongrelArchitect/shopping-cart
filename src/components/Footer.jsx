import React from 'react';

export default function Footer() {
  return (
    <div className="footer">
      <ul>
        <li>
          Designed by
          {' '}
          <a
            target="_blank"
            href="http://www.seanericthomas.com"
            rel="noreferrer"
          >
            Sean Eric Thomas
          </a>
          .
        </li>
        <li>
          Products from
          {' '}
          <a target="_blank" href="https://fakestoreapi.com/" rel="noreferrer">
            Fake Store API
          </a>
          , image from
          {' '}
          <a
            target="_blank"
            href="https://unsplash.com/photos/c9FQyqIECds"
            rel="noreferrer"
          >
            Unsplash
          </a>
          .
        </li>
      </ul>
    </div>
  );
}
