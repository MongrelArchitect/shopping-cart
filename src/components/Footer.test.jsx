import React from 'react';
import { render } from '@testing-library/react';

import Footer from './Footer';

describe('Footer component', () => {
  it('renders correctly', () => {
    const { container } = render(<Footer />);
    expect(container).toMatchSnapshot();
  });
});
