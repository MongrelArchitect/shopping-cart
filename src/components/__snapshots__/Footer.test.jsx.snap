// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component renders correctly 1`] = `
<div>
  <div
    class="footer"
  >
    <ul>
      <li>
        Designed by
         
        <a
          href="http://www.seanericthomas.com"
          rel="noreferrer"
          target="_blank"
        >
          Sean Eric Thomas
        </a>
        .
      </li>
      <li>
        Products from
         
        <a
          href="https://fakestoreapi.com/"
          rel="noreferrer"
          target="_blank"
        >
          Fake Store API
        </a>
        , image from
         
        <a
          href="https://unsplash.com/photos/c9FQyqIECds"
          rel="noreferrer"
          target="_blank"
        >
          Unsplash
        </a>
        .
      </li>
    </ul>
  </div>
</div>
`;
